python/cucim/ci/bootstrap.py

```diff
@@ -1,16 +1,11 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import
-from __future__ import print_function
-from __future__ import unicode_literals
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
We shouldn't need the `__future__` imports here since we are Python 3.x only

Reply:
Thanks Greg!
oh, the file is not used in RAPIDS CI/CD but will remove it.